.gitlab-ci.yml

```diff
@@ -422,13 +422,14 @@ test-rust-stable:
   image: ethcore/rust:stable
   before_script:
     - git submodule update --init --recursive
+    - export RUST_FILES_MODIFIED=$(git --no-pager diff --name-only $CI_BUILD_REF^ $CI_BUILD_REF | grep \.rs | wc -l)
```

Comment:
There a few other extensions that need to trigger rusts tests. e.g. .json, .toml.
Better just check if any of the non-js files have been modified with something like `grep -v \.js` instead of `grep -j \.rs`